File: 백준/Bronze/5063. TGN/TGN.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
	public static void main(String[] args) {
		try {
			BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
			
			int testCase = Integer.parseInt(br.readLine());
			
			for (int i=0; i<testCase; i++) {
				String data[] = br.readLine().split(" ");
				int r = Integer.parseInt(data[0]);
				int e = Integer.parseInt(data[1]);
				int c = Integer.parseInt(data[2]);
				
				int res = e-c;
				
				if(r>res) {
					System.out.println("do not advertise");
				}else if (res>r) {
					System.out.println("advertise");
				}else {
					System.out.println("does not matter");
				}
			}
			
		}catch(IOException e) {
			System.out.println("error");
		}
	}
}

File: 백준/Bronze/5063. TGN/TGN.py
t = int(input())

for _ in range(t):
    r, e, c = map(int, input().split())

    test_case = e - r - c

    if test_case > 0:
        print("advertise")
    elif test_case == 0:
        print("does not matter")
    else:
        print("do not advertise")